internal/adapters/http/handler.go

```diff
@@ -0,0 +1,51 @@
+package http
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/streadway/amqp"
+
+	"ecommerce_order/internal/adapters/rabbitmq"
+	"ecommerce_order/internal/application"
+	"ecommerce_order/internal/domain"
+)
+
+func RegisterRoutes(r *gin.Engine) {
+
+	amqpURL := os.Getenv("RABBITMQ_URL")
+	if amqpURL == "" {
+		log.Fatal("RABBITMQ_URL not configured in .env file")
+	}
+
+
+	conn, err := amqp.Dial(amqpURL)
+	if err != nil {
+		log.Fatal("Eror to connect RabbitMQ:", err)

```

Comment:
substitua isso aqui por algo como:


```suggestion
// internal/config/config.go
package config

import (
	"log"
	"os"
)

type Config struct {
	RabbitURL string
}

func Load() *Config {
	return &Config{
		RabbitURL: os.Getenv("RABBITMQ_URL"),
	}
}

```